src/components/Table/Table.tsx

```diff
@@ -27,11 +32,21 @@ export interface TableProps {
   columnHeadersBackgroundColor?: string;
   /** Hex value to set the text color of the column headers. */
   columnHeadersTextColor?: string;
+  /** Array of style objects used to set custom width values for the table
+   * columns. Will accept "width" and "maxWidth" attributes. */
```

Comment:
are `width` and `maxWidth` examples or the only expected values that are passed?